cmd/prysmctl/withdrawal/withdrawal.go

```diff
@@ -0,0 +1,188 @@
+	defer span.End()
+	beaconNodeHost := "http://localhost:3500"
+	if c.String(BeaconHostFlag.Name) != "" {
+		beaconNodeHost = c.String(BeaconHostFlag.Name)
+	}
+	u, err := url.ParseRequestURI(beaconNodeHost)
+	if err != nil {
+		return errors.Wrap(err, "invalid format, unable to parse url")
+	}
+	if u.Scheme == "" || u.Host == "" {
+		return fmt.Errorf("provided url %s is not in the format of http(s)://host:port", beaconNodeHost)
+	}
+	foundFilePaths, err := findWithdrawalFiles(c.String(FileFlag.Name))
+	if err != nil {
+		return errors.Wrap(err, "failed to find withdrawal files")
+	}
+	if len(foundFilePaths) == 0 {
+		return errors.New("no compatible files were found")
+	}
+	au := aurora.NewAurora(true)
+	fmt.Println(au.Red("===============IMPORTANT==============="))
+	if !c.Bool(SkipPromptsFlag.Name) {
+		fmt.Println(au.Red("Please read the following carefully"))
+	}
+	fmt.Print("This action will allow the partial withdraw of amounts over the 32 staked eth in your active validator balance. \n" +
+		"You will also be entitled to the full withdrawal of the entire validator balance if your validator has exited. \n" +
+		"The partial or full withdrawal of the validator balance may require several days of processing. \n" +
+		"Please navigate to our website and make sure you understand the full implications of setting your withdrawal address. \n")
+	fmt.Println(au.Red("THIS ACTION WILL NOT BE REVERSIBLE ONCE INCLUDED. "))
+	fmt.Println(au.Red("You will NOT be able to change the address again once changed. "))
+
+	setWithdrawalAddressJsons := make([]*apimiddleware.SignedBLSToExecutionChangeJson, 0)
+	for _, foundFilePath := range foundFilePaths {
+		b, err := os.ReadFile(filepath.Clean(foundFilePath))
+		if err != nil {
+			return errors.Wrap(err, "failed to open file")
+		}
+		if string(b)[0:1] != "[" {
+			log.Warnf("provided file: %s, is not a list \n", foundFilePath)
+			continue
+		}
+		var to []*apimiddleware.SignedBLSToExecutionChangeJson
+		if err := json.Unmarshal(b, &to); err != nil {
+			return errors.Wrap(err, "failed to unmarshal file")
+		}
+		setWithdrawalAddressJsons = append(setWithdrawalAddressJsons, to...)
+	}
+	if len(setWithdrawalAddressJsons) == 0 {
+		return errors.New("the list of signed requests is empty")
+	}
+	if !c.Bool(SkipPromptsFlag.Name) {
+		for _, jsonOb := range setWithdrawalAddressJsons {
+			if err := verifyWithdrawalCertainty(r, jsonOb); err != nil {
+				return err
+			}
+		}
+	}
+	return callWithdrawalEndpoint(ctx, beaconNodeHost, setWithdrawalAddressJsons)
+}
+
+func verifyWithdrawalCertainty(r io.Reader, request *apimiddleware.SignedBLSToExecutionChangeJson) error {
+	au := aurora.NewAurora(true)
+	withdrawalConfirmation := request.Message.ToExecutionAddress
+	fmt.Println(au.Red("===================================="))
+	fmt.Println("YOU ARE ATTEMPTING TO CHANGE THE BLS WITHDRAWAL(" + au.Red(request.Message.FromBLSPubkey).String() + ") ADDRESS " +
```

Comment:
I think we don't need to include the withdrawal address here. This is something that is not known to 99.99% of users and will make this message less readable. They need to see the execution address and the validator index and it would be easier to check validity.

Reply:
can reword this.